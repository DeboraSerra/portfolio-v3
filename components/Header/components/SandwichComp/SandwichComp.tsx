import { NextPage } from "next";

import * as S from './SandwichComp.styled'

interface Props {}

const SandwichComp: NextPage<Props> = () => {
  return <S.Main></S.Main>
}

export default SandwichComp 