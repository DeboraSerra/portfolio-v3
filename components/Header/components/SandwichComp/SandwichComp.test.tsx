import { render } from "@testing-library/react";
import SandwichComp from "./SandwichComp";

describe("Test the SandwichComp component", () => {
  it('Should match the snapshot', () => {
    const { asFragment } = render(<SandwichComp />)
    expect(asFragment()).toMatchSnapshot();
  })
})