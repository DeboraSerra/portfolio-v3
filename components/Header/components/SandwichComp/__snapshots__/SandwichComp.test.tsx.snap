// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test the SandwichComp component Should match the snapshot 1`] = `
<DocumentFragment>
  <div
    class=""
  />
</DocumentFragment>
`;
